client/flutter/lib/l10n/intl_hi.arb

```diff
@@ -0,0 +1,26 @@
+{
+    "worldHealthOrganization": "विश्व स्वास्थ्य संगठन",
+    "learnMore": "और अधिक जानें|",
+    "protectYourself": "अपनी रक्षा कीजिये",
+    "checkYourHealth": "अपने स्वास्थ्य की जाँच करें",
+    "localMaps": "स्थानीय मानचित्र",
+    "shareTheApp": "ऍप को बाँटे ",
+    "healthCheck": "स्वास्थ्य जांच",
+    "feelingDistressed": "परेशान है?",
+    "washHands": "अपने हाथों को धोएं",
+    "cougningAndSneezing": "खाँसते और छींकते हुए मुंह और नाक को लचीली कोहनी या ऊतक से ढक ले",
```

Comment:
coughingAndSneezing

Reply:
It's look good

Reply:
@sharadcode changing the key to coughingAndSneezing only for Hindi translations may break the app when the locale is hi.

You can create a new issue for the typo.